cmd/flux/create_source_git_test.go

```diff
@@ -87,7 +87,7 @@ func (r *reconciler) conditionFunc() (bool, error) {
 
 func TestCreateSourceGit(t *testing.T) {
 	// Default command used for multiple tests
-	var command = "create source git podinfo --url=https://github.com/stefanprodan/podinfo --branch=master --timeout=" + testTimeout.String()
+	var command = "create source git podinfo --url=https://github.com/stefanprodan/podinfo --branch=master --ignore-path .cosign,non-existent-dir/ --timeout=" + testTimeout.String()
```

Comment:
Please add an assertion of the `.spec.ignore` field here.

Reply:
i tried to add a `--export` flag to command and add a new case here but since these unit test cases deal with adding a status to the object and then verifying it, they always deal with a valid (and dynamic) namespace. i couldn't assert with a dynamic namespace, so i created a new test case where a gitrepository resource is simply exported (printed to stdout) and then asserted against a file.